coding-the-legislation/legislation_parameters.md

```diff
@@ -1,93 +1,147 @@
 # Legislation parameters
 
-## Definition
 
-The legislation parameters are stored in XML files in the directory `parameters` ([example in OpenFisca-France](https://github.com/openfisca/openfisca-france/blob/master/openfisca_france/parameters)).
+## Hierarchical structure
 
-Those files contain parameters which can be simple parameters (`<CODE>` tags) or scales (`<BAREME>` tags).
+The parameters are organized with a [tree structure](https://en.wikipedia.org/wiki/Tree_structure). Each node of the tree can contain scales, parameters or other nodes ("children nodes").
```

Comment:
Could you explain a little bit more about the link between the structure (node, scale, children) and the Legislation parameters ?